### PR TITLE
improve-9-3d-charts

### PR DESCRIPTION
**Pull request by Vazonik**

### CODE REVIEW

**Review by mfilipiec — APPROVED** [review]
LGTM

File: highcharts-api/highcharts/11-3d-funnels/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Exercise 11 - 3D Funnels
+
+Create a 3D chart composition, like in the image below, using two series:
+`funnel3d` and `pyramid3d`.
```
Comment (pawellysy) [inline highcharts-api/highcharts/11-3d-funnels/README.md:4]: It sounds like the student was supposed to create some module, since there is even a module named `3dFunnelComposition` etc. I'd rephrase that, e.g. 
```Create a combination of 3D series types in the chart. ```

**Review by pawellysy — APPROVED** [review]
